Misc/NEWS.d/next/Core_and_Builtins/2025-12-31-17-38-33.gh-issue-143308.lY8UCR.rst

```diff
@@ -0,0 +1,3 @@
+:mod:`pickle`: fix use-after-free crashes when a :class:`~pickle.PickleBuffer`
```

Comment:
Ah I actually didn't see, but could you move this NEWS entry from "Core_and_Builtins" to "Library"? The patch affects pickle which is a non-builtin component (core and builtins is reserved for built-in types and the interpreter itself, not what we call "extension modules" in Modules/ or in pure Python modules in Lib/). TiA.

Reply:
Thanks for the guidance, moved!